talentprotocol/web-dapp
# feat: partnership page improvements

## Summary

This PR;
- adds `banner_data`, `location`, `button_name` and `button_url` to `partnerships`
- creates a rake task that receives a JSON URL, an invite_code, a user_id and the max invite uses to create a partnership
- alters the discovery page to reflect the [Figma designs](https://www.figma.com/file/Ovbho8ofi2uFW6dOxheWXx/Talent-Protocolo---Dapp?node-id=0%3A1)

## Screenshots

Partnership Discovery

<img width="1440" alt="Screenshot 2022-10-18 at 13 41 50" src="https://user-images.githubusercontent.com/33064059/196432911-b4868313-ba16-4276-89f3-859ee69ef873.png">

Non-Partnership Discovery

<img width="1432" alt="Screenshot 2022-10-18 at 13 41 59" src="https://user-images.githubusercontent.com/33064059/196432930-09907b76-1710-4d41-b4a2-6accc2c2ef53.png">


## Notion link

https://talentprotocol.notion.site/Improvements-to-Organizations-page-99a1c3c36b774f0aa0c26a46e44bcb28

## How to test?

See `lib/tasks/partnerships/utrust.json` for an example JSON structure to create a partnership.
